Makefile

```diff
@@ -1049,6 +1048,8 @@ $(TARBALL): release-only $(NODE_EXE) doc
 	$(RM) -r $(TARNAME)/tools/node_modules
 	$(RM) -r $(TARNAME)/tools/osx-*
 	$(RM) -r $(TARNAME)/tools/osx-pkg.pmdoc
+	find $(TARNAME)/deps/v8/test/* -type d ! -regex '.*/test/torque$$' | xargs $(RM) -r
```

Comment:
What's this one doing? I'm trying it out on a mac (where we've had `find` problems before) and the regex doesn't seem to be adding anything. If I remove `! -regex '.*/test/torque$$'` it returns the same results as with it.

The next line, on the other hand, excludes `./deps/v8/test/torque/test-torque.tq` and `./deps/v8/test/torque/OWNERS` which are matched without the `! -regex`

Reply:
This line recursively deletes all subdirectories of `deps/v8/test` except `deps/v8/test/torque` (i.e. the regex prevents that directory from being removed). The second line removes the leftover files in `deps/v8/test` (we cannot recursively remove `deps/v8/test` as that would also remove the `torque` subdirectory).

If the regex isn't doing anything on you mac maybe try with a single `$` (i.e. `! -regex '.*/test/torque$'`)? I originally only had the one `$` there as that is what I had working in a bash prompt but it appeared to need escaping in the makefile at least on AIX and the Linux system I tried this on. 

I actually am thinking that this should probably be moved out of the makefile into a callable script (either shell or Python (it would be similar to the existing `tools/install.py`)), but that's a separate discussion/pull request.

Reply:
Ah, so this'll get stripped out when running from Makefile, goodo. I was trying it on the command line which explains the difference.

```sh
$ find ./deps/v8/test/* -type d ! -regex '.*/test/torque$$' > /tmp/f1
$ find ./deps/v8/test/* -type d > /tmp/f1
$ diff /tmp/f1 /tmp/f2
$ find ./deps/v8/test/* -type d ! -regex '.*/test/torque$' > /tmp/f1
$ diff /tmp/f1 /tmp/f2
191a192
> ./deps/v8/test/torque
$
```